tools/doc/html.js

```diff
@@ -165,10 +175,32 @@ function parseLists(input) {
 }
 
 
+// Syscalls which appear in the docs, but which only exist in BSD / OSX
+var BSD_ONLY_SYSCALLS = ['lchmod'];
```

Comment:
Yeah, I would think that utilizing a `Set` or even an object would be faster here (assuming we get more items added to here as well.

```js
var BSD_ONLY_SYSCALLS = new Set(['lchmod']);

...
if (BSD_ONLY_SYSCALLS.has(name)( {
  ...
} else {
...
}
```